packages/cubejs-query-orchestrator/test/unit/QueryOrchestrator.test.js

```diff
@@ -1575,4 +1594,27 @@ describe('QueryOrchestrator', () => {
     await Promise.all(promises);
     expect(mockDriver.tables).toContainEqual(expect.stringMatching(/orders_delay/));
   });
+
+  test('fetch table schema', async () => {
```

Comment:
Hey! I'm wondering what you are trying to test here? This test implementation looks senseless to me: you mock the method with the constant and then compare it with the same constant.